src/video/x11/SDL_x11toolkit.h

```diff
@@ -175,7 +175,8 @@ typedef struct SDL_ToolkitControlX11
     bool dynamic;
     bool is_default_enter;
     bool is_default_esc;
-
+	bool skip_size;
```

Comment:
The default value for this is true. Maybe invert it and maybe rename it to be more specific like `include_padding`?